ee/tabby-webserver/src/service/auth.rs

```diff
@@ -218,6 +220,53 @@ impl AuthenticationService for AuthenticationServiceImpl {
         Ok(resp)
     }
 
+    async fn request_password_reset_email(&self, email: String) -> Result<String> {
```

Comment:
```suggestion
    async fn request_password_reset_email(&self, email: String) -> Result<()> {
```

If you need this from test, consider read it from email api.